process-migration-service/pom.xml

```diff
@@ -43,7 +43,8 @@
     <kie.server.password>usetheforce123@</kie.server.password>
 
     <!-- PIM service configuration -->
-    <pim.server.http>http://${container.hostname}:8180/</pim.server.http>
+    <pim.server.http.port>18080</pim.server.http.port>
+    <pim.server.http>http://${container.hostname}:${pim.server.http.port}/</pim.server.http>
```

Comment:
This is just to make tests more stable? As I think that the KIE Server runs on 8080, so the conflict shouldn't be happening. Although from other tests/modules it can...

Reply:
the build log also shows some cargo problems with starting the service or something before ProcessMigrationIntegrationTest

Reply:
Yeah, that would signal that something before wasn't properly stopped. But still a good precaution to use a very different port.

Reply:
I can't explain why I can't reproduce it locally. My only guess is that maybe some other service is using port 8180 or that certain ports are not allowed.

Reply:
Yes, I think it can also happen if the jenkins slave is reused after the previous run wasn't successfully finished etc.